### PR TITLE
Enable post-search-results everywhere

### PR DESCRIPTION
**Pull request by vforge**
## Description

- Enable post-search-results everywhere*
- Fix escaped `&hellip;`
- Fix styles for "SEE MORE ->"

*) if the community has discussions enabled OR this is the cross wiki search

## Reviewers

@Wikia/cake

### CODE REVIEW

**Review by JoshuaRogan — APPROVED** [review]

File: app/styles/component/_post-search-results.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
   padding-bottom: 10px;
 
   &__header {
+    display: flex;
```
Comment (JoshuaRogan) [inline app/styles/component/_post-search-results.scss:7]: Any support issues with this?

File: app/templates/components/post-search-results-item.hbs [hunk 2]
```diff
@@ -6,7 +6,7 @@
   class="post-search-results-item__title wds-leading-tight"
   onclick={{action "trackClick" postIndex}}
 >
-  {{getTitle}}
+  {{{getTitle}}}
```
Comment (JoshuaRogan) [inline app/templates/components/post-search-results-item.hbs:9]: Why the extra `{}`?

File: app/styles/component/_post-search-results.scss (hunk 1 above)
Comment (vforge) [reply app/styles/component/_post-search-results.scss:7]: nothing that we should care about :)

File: app/templates/components/post-search-results-item.hbs (hunk 2 above)
Comment (vforge) [reply app/templates/components/post-search-results-item.hbs:9]: un-escape the string

**Pull request merged**